src/button-group.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {createUseStyles} from './jss';
+import {Inline} from '.';
+import {ButtonLink, ButtonLinkProps, ButtonPrimary, ButtonProps, ButtonSecondary} from './button';
+
+const buttonLayoutSpacing = 16;
+const buttonLinkPadding = 6;
+
+const useStyles = createUseStyles(() => ({
+    buttonsContainer: {
+        display: 'flex',
+        flexWrap: 'wrap',
+        alignItems: 'center',
+    },
+    fullWidthLink: {
+        marginLeft: -buttonLinkPadding,
+        width: '100%',
+        marginTop: buttonLayoutSpacing,
+    },
+}));
+
+export interface ButtonGroupProps {
+    primaryButton?: React.ReactElement<ButtonProps, typeof ButtonPrimary>;
+    secondaryButton?: React.ReactElement<ButtonProps, typeof ButtonSecondary>;
```

Comment:
I'm checking if I shouldn't restrict these types to primary and secondary buttons since I see some usages in which the secondary button is configured with a ButtonPrimary:
 https://github.com/Telefonica/webapp/blob/master/web/src/pages/tariff/o2-de/components/tariff.js#L37-L40